### ISSUE TITLE
2012-11-06 election has zero votes?

### ISSUE THREAD

**Issue by jeremyjbowers** [newsdev/nyt-ap-elections#104]
Probably shouldn't fail like this:

```
2015-11-19 16:22:23,859 (INFO) elex (v0.0.39) : Getting reporting units for election 2012-11-06
Traceback (most recent call last):
  File "/Users/jbowers/.virtualenvs/fakelections/bin/elex", line 9, in <module>
    load_entry_point('nyt-ap-elections==0.0.39', 'console_scripts', 'elex')()
  File "/Users/jbowers/.virtualenvs/fakelections/src/nyt-ap-elections/elex/cli/app.py", line 156, in main
    app.run()
  File "/Users/jbowers/.virtualenvs/fakelections/lib/python2.7/site-packages/cement/core/foundation.py", line 764, in run
    self.controller._dispatch()
  File "/Users/jbowers/.virtualenvs/fakelections/lib/python2.7/site-packages/cement/core/controller.py", line 472, in _dispatch
    return func()
  File "/Users/jbowers/.virtualenvs/fakelections/src/nyt-ap-elections/elex/cli/decorators.py", line 13, in decorated
    return fn(self)
  File "/Users/jbowers/.virtualenvs/fakelections/src/nyt-ap-elections/elex/cli/app.py", line 74, in reporting_units
    self.app.render(self.app.election.reporting_units)
  File "/Users/jbowers/.virtualenvs/fakelections/src/nyt-ap-elections/elex/parser/api.py", line 631, in reporting_units
    race_objs = self.get_race_objects(raw_races)
  File "/Users/jbowers/.virtualenvs/fakelections/src/nyt-ap-elections/elex/parser/api.py", line 571, in get_race_objects
    return [Race(**r) for r in parsed_json['races']]
  File "/Users/jbowers/.virtualenvs/fakelections/src/nyt-ap-elections/elex/parser/api.py", line 422, in __init__
    self.set_reportingunits()
  File "/Users/jbowers/.virtualenvs/fakelections/src/nyt-ap-elections/elex/parser/api.py", line 50, in set_reportingunits
    obj = ReportingUnit(**reportingunit_dict)
  File "/Users/jbowers/.virtualenvs/fakelections/src/nyt-ap-elections/elex/parser/api.py", line 339, in __init__
    self.set_candidate_votepct()
  File "/Users/jbowers/.virtualenvs/fakelections/src/nyt-ap-elections/elex/parser/api.py", line 357, in set_candidate_votepct
    c.votepct = float(c.votecount) / float(self.votecount)
ZeroDivisionError: float division by zero
```

**Comment by jeremyjbowers** [comment]
Should trap for divide by zero anyway, because of reasons.

**Comment by eads** [comment]
And also write a test!!

**Comment by jeremyjbowers** [comment]
Test here: 4ef35bcd398493991c2e621e2490a1dfea6d0cfc
Fix here: 5e214e59ae2d484de18e4cb6af379a6f7ba4248f
